test/git.test.ts

```diff
@@ -0,0 +1,52 @@
+import { fromUrl } from 'hosted-git-info';
+import { getTarget } from '../lib/git';
```

Comment:
This test looks good but I noticed that we generated all the tests in this plugin in the following format `name.test.ts` instead of `name.spec.ts` as it should be for a Jest test, we can `rename/refactor` in a side pr

Reply:
jest supports either - lets defer that to a refactor PR if you feel it's important

Reply:
Yeah no worries as I mentioned above ☝️  `we can rename/refactor in a side pr` I just wanted to keep the congruence `.spec` jest and `.test` tap ^^